lib/hippie/client.js

```diff
@@ -13,6 +13,7 @@ var http = require('http');
 var serializers = require('./serializers');
 var parsers = require('./parsers');
 var expect = require('./expect');
+var Promise = require('es6-promise').Promise;
```

Comment:
`3` is an option. Could also have have `hippie.Promise` which is `global.Promise || require('es6-promise')`. Then all you have to do is either have you're preferred promise lib mounted to global or you can overwrite `hippie.Promise`.

I would prefer that if there is a global promise lib to just use that, as then using babel or Node 4.x requires no additional configuration.